recipe/meta.yaml

```diff
@@ -26,36 +23,41 @@ requirements:
     - {{ compiler('cxx') }}
   host:
     - python
-    - cython
+    - glibmm
+    - glib 2.56
+    - jinja2
     - pandas
+    - cython 0.26
+    - coincbc 2.9
```

Comment:
Since you are using an older `coincbc`, that older version needs to be rebuilt for the new blas scheme.